catalog/core/catalog-core-impl/pubsub/src/main/java/ddf/catalog/pubsub/criteria/temporal/TemporalEvaluationCriteriaImpl.java

```diff
@@ -25,21 +25,36 @@
 
     public TemporalEvaluationCriteriaImpl(Date end, Date start, Date input) {
         super();
-        this.end = end;
-        this.start = start;
-        this.input = input;
+        if(end != null) {
```

Comment:
Can we do a quick reformat of the lines that were changed? Looks like we're missing spaces between `if` and `(` in a bunch of places.